tests/test_output/1so_apps.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
I think 100% object-matching like this will cause a lot of build instability. If you make any changes in splunk-ansible, this will break. If we want to do exact matches like this, it should be done closer to the source of the code generating this blob.

On the other-hand, I get the intention for checking that the inventory + vars line up. For each of the scenarios, it's best to make sure certain keys have well-defined values (ex. in this case, make sure `json["all"]["vars"]["splunk"]["apps_location"]` is defined and has the value you expect) vs. doing a full comparison of everything.